src/client.ts

```diff
@@ -19,7 +19,9 @@ const send = (options: ICourierClientConfiguration) => {
       data: params.data,
       event: params.eventId,
       profile: params.profile,
-      recipient: params.recipientId
+      recipient: params.recipientId,
+      preferences: params.preferences,
```

Comment:
build is failing here: `The key 'preferences' is not sorted alphabetically`

Reply:
Interesting. I wondered why they were in that order.

Reply:
some of us use `tslint` to automatically show these issues in vscode if you are interested: https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin

Reply:
Thank you. That helps.